### DISCUSSION TITLE
download open3d on python

### DISCUSSION THREAD

**Discussion by cakemout** [community/community#88440] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I downloaded open3d on Python, and the Python version says 3.10.11, and when I pip install open3d, I got this error: "ERROR: Could not find a version that satisfies the requirement open3d (from versions: none) ERROR: No matching distribution found for open3d"
So how does Python 3.10.11 download open3d?

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @cakemout! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
